gnome-initial-setup/gnome-initial-setup-copy-worker.c

```diff
@@ -55,7 +55,7 @@ move_file_from_homedir (GFile       *src_base,
 
   g_file_make_directory_with_parents (dest_parent, NULL, NULL);
 
-  if (!g_file_move (src, dest, G_FILE_COPY_NONE,
+  if (!g_file_copy (src, dest, G_FILE_COPY_NONE,
```

Comment:
Hmm, so effectively this only works because the initial setup homedir is still around (we haven't rebooted and initial setup did run this boot), and the copy worker runs every login.

However I don't like the solution and I think it will have unintended side-effects; for instance if you go through the FBE normally create an user, log out and then log in again, the settings will be again overwritten unless you reboot.

In reality, when we enter the demo mode there are no settings that we care about that we aren't explicitly setting in the `eos-demo-mode` script; so instead of changing the settings in the initial setup session and copy the database over, we could simply set them in the actual demo mode session every time we log in.